chatcommands.py

```diff
@@ -548,15 +548,21 @@ def info():
 
 # noinspection PyIncorrectDocstring
 @command()
```

Comment:
You're [going to need](https://stackoverflow.com/c/charcoal/a/59/1) `whole_msg=True` in the decorator here.